CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ### New and Improved
 
+## 0.19.2 (2025/05/08)
+
 * Adds support to parse User-Agent headers and emit them in telemetry events
   ([PR](https://github.com/hashicorp/boundary/pull/5645)).
```

Comment:
I think this change will go into the next release, not this release. Could you move it to `next`?